extensions/amp-subscriptions-google/0.1/test-e2e/test-amp-subscriptions-google.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'amp-subscriptions-google',
+  {
+    testUrl:
+      'http://localhost:8000/test/manual/amp-subscriptions-google/swg.amp.html',
```

Comment:
Currently other components use the path `test/fixtures/e2e/amp-subscriptions-google/swg.html`

Reply:
I'll move it over. Thanks!